quint/src/cliCommands.ts

```diff
@@ -283,7 +283,7 @@ export function runTests(prev: TypecheckedStage): CLIProcedure<TestedStage> {
       out(`\n  ${mainName}`)
     }
 
-    const matchFun =
+    const _matchFun =
       (n: string): boolean => isMatchingTest(prev.args.match, n)
 
```

Comment:
Seems like we can get rid of this

```suggestion
```

Reply:
Hmm. I thought I had gotten rid of this in https://github.com/informalsystems/quint/pull/774/commits/1823c5ba0010b83ad30d2f504b09da40fc1f51ad  

I wonder if I undid something before commiting? Thanks!